commons/src/main/java/io/aiven/kafka/tieredstorage/commons/UniversalRemoteStorageManagerConfig.java

```diff
@@ -131,11 +134,18 @@ public class UniversalRemoteStorageManagerConfig extends AbstractConfig {
         );
 
         CONFIG.define(
-            COMPRESSION_CONFIG,
+            COMPRESSION_ENABLED_CONFIG,
+            ConfigDef.Type.BOOLEAN,
+            false,
+            ConfigDef.Importance.HIGH,
+            COMPRESSION_ENABLED_DOC
+        );
+        CONFIG.define(
+            COMPRESSION_HEURISTIC_CONFIG,
             ConfigDef.Type.BOOLEAN,
             false,
             ConfigDef.Importance.HIGH,
-            COMPRESSION_DOC
+            COMPRESSION_HEURISTIC_DOC
         );
```

Comment:
I think these two need a joint validation, e.g. enabling the heuristic without enabling compression in general maybe doesn't make sense, WDYT?